ethereum/core/src/test/java/org/hyperledger/besu/ethereum/worldstate/MarkSweepPrunerTest.java

```diff
@@ -147,7 +147,8 @@ public void sweepBefore_shouldSweepStateRootFirst() {
     stateRoots.forEach(
         stateRoot -> {
           final InOrder thisRootsOrdering = inOrder(hashValueStore, worldStateStorage);
-          thisRootsOrdering.verify(hashValueStore).remove(stateRoot);
+          // disable this validation since the underlying storage no longer directly uses .remove()
+          // thisRootsOrdering.verify(hashValueStore).remove(stateRoot);
```

Comment:
looking for feedback here - do we care to jump through hoops to keep the pruning tests passing when pruning is a deprecated feature?